src/main/java/edu/ie3/datamodel/models/input/container/ThermalGrid.java

```diff
@@ -48,8 +58,10 @@ public String toString() {
         + bus
         + ", #houses="
         + houses.size()
-        + ", #storages="
-        + storages.size()
+        + ", #heatStorages="
```

Comment:
Why do we add a `#` here?

Reply:
Because its the number of houses, storages ... the # indicates number of...